cloud/endagaweb/templates/dashboard/user_management/add.html

```diff
@@ -0,0 +1,354 @@
+        {% get_obj_perms request.user for network as 'user_permission' %}
+        <div class="container col-sm-10">
+            {% for message in messages %}
+            <div class="message alert alert-{{message.tags}}">
+                <a href="#" class="close" data-dismiss="alert">&times;</a>
+                {{message}}
+            </div>
+            {% endfor %}
+            {% if 'user_management' in user_permission %}
+            <form action="/dashboard/user/management" id="user_form" name="user_form" method="POST"
+                      data-toggle="validator" onsubmit="return createUser();"> {% csrf_token %}
+                    <!--Role-->
+                    <div class="form-group row" data-toggle="tooltip" title="Select the user role" required>
+                        <div class="col-xs-5">
+                            <label>Role <span class="aseriskField">*</span></label>
+                            <div class="input-group col-xs-8">
+                                <div class="input-group-addon">
+                                    <i class="glyphicon glyphicon-user"></i>
+                                </div>
+                                <select name="role" id="role" class="form-control" onchange="changeRole(this)" required>
+                                    <option value="">-Select role-</option>
+                                    {% for values in roles %}
+                                    <option value="{{ values }}">{{ values }}</option>
+                                    {% endfor %}
+                                </select>
+
+                            </div>
+                        </div>
+                    </div><!--Role Ends-->
+
+                    <!--Email-->
+                    <div class="form-group row">
+                        <div class="col-xs-5" data-toggle="tooltip" data-placement="left"
+                             title="User email to Add or Edit">
+                            <label for="email" id="ariaLable" class="col-3 col-form-label">Email <span class="aseriskField">*</span></label>
+                            <div class="input-group col-xs-8">
+                                <div class="input-group-addon">
+                                    <i class="glyphicon glyphicon-envelope"></i>
+                                </div>
+                                <input type="hidden" name="valid_email" id="valid_email" value="0"/>
+                                <input class="form-control" type="text" id="email" name="email" placeholder="abc@yourdomain.com" required/>
+                            </div>
+                            <span class="msg error">Not a valid email address</span>
+                             <span class="msg exists"><a href="javascript:editUser();">Edit user</a></span>
+                            <span class="msg success"></span>
+                        </div>
+                    </div><!--Email Ends-->
+
+                    <!--Password-->
```

Comment:
Password is kept temporarily for testing purpose.
If we mention existing user in email field , an edit option will be shown just below email for editing the user.
Please find attached snapshots
![editscreen](https://user-images.githubusercontent.com/27676750/28162426-939cdeac-67e4-11e7-81ec-eb9f56324450.png)
![existinguser](https://user-images.githubusercontent.com/27676750/28162427-939eee18-67e4-11e7-9f0e-3ea681f103ff.png)